applications/app/views/fragments/audioSeriesHead.scala.html

```diff
@@ -55,7 +57,10 @@ <h1 class="flagship-audio__title">
 
 <div id="flagship-audio__mobile-only">
     <ul class="flagship-audio__content-buttons">
-        <p class="flagship-audio__subscribe-label">Subscribe for free</p>
+        @if(!subscribeLinks.isEmpty){
+            <p class="flagship-audio__subscribe-label">Subscribe for free</p>
+        }
+
         @for((name: String, link : String) <- subscribeLinks) {
```

Comment:
might as well move this inside the above `if` block

Reply:
okay